tools/webots.py

```diff
@@ -25,6 +25,10 @@
 ROBOCUP_IMAGE_TAG = "robocup2021"  # Submitted in our team_config.json, shouldn't be changed here unless changed there
 ROBOCUP_IMAGE_REGISTRY = "079967072104.dkr.ecr.us-east-2.amazonaws.com/robocup-vhsc-nubots"  # Provided by the TC
 
+# Webots team details
+NUBOTS_TEAM_ID = 12
```

Comment:
I am curious what the significance of this is

Reply:
This is our official team ID from RoboCup 2021, specified in the GameController repo.
13 is just the number after 12 :stuck_out_tongue:

Reply:
I think it'd be worth adding that as a comment tbh. I feel like I could see someone in a couple of years having a lot of trouble trying to work out that :sweat_smile:

Reply:
We get assigned a different team id each robocup afaik. I got `12` from what is currently set in [GlobalConfig.yaml](https://github.com/NUbots/NUbots/blob/main/module/support/configuration/GlobalConfig/data/config/GlobalConfig.yaml). It doesnt matter what the alt team id is, as long as it is different

Reply:
Added a relevant comment and removed the alt team ID in favour of just incrementing the NUbots team ID for the other team ID.